db/seedfiles/translations/en/cca/insured.rb

```diff
@@ -347,6 +347,8 @@
   :'en.your_coverage_start_date' => "Your coverage start date",
   :'en.standard_plan' => "Standard Plan",
   :'en.insured.plan_shoppings.standard_plan_title_info' => "Each health insurance company offers a standard plan at each metal level. Benefits and cost-sharing are the same among standard plans of the same metal level, but monthly premiums and provider network options may be different. This makes it easier for consumers to compare plans at the same metal level and choose what’s best for them.",
+  :'en.premium_value_plan' => "Premium Value Plan",
+  :'en.insured.plan_shoppings.pvp_title_info' => "Exchange offers a Premium value plan at each rating area.",
```

Comment:
Is this perhaps placeholder text? It seems a lot briefer than the tooltip for `standard_plan_title_info`.